### PR TITLE
Fix warn found by https://github.com/apache/incubator-nuttx/pull/7391

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
Found by https://github.com/apache/incubator-nuttx/pull/7391

## Impact
Minor

## Testing
Pass CI

### CODE REVIEW

**Review by pkarashchenko — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
@pkarashchenko could you merge this simple change? I can continue on https://github.com/apache/incubator-nuttx/pull/ only after this patch get merged.

**Comment by pkarashchenko** [comment]
Yes. Just waiting for CI to pass

**Pull request merged**
